db/migrate/20160219145341_create_answers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration
+  def change
+     create_table :answers do |t|
+      t.references :question, null: false
+      t.references :user, null: false
+      t.references :survey, null: false
+      t.references :choice, null: false
```

Comment:
Likewise... if you know the choice, you also know the question and the survey...